<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectMainForm</class>
 <widget class="QMainWindow" name="ProjectMainForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>948</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>8</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="locale">
       <locale language="Vietnamese" country="VietNam"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Modules</string>
    </property>
    <addaction name="actionImport_Module"/>
    <addaction name="actionRemove_Module"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAssets">
    <property name="title">
     <string>Assets</string>
    </property>
    <addaction name="actionAsset_Tracking"/>
    <addaction name="actionAsset_Browser"/>
   </widget>
   <widget class="QMenu" name="menuChecking_Tools">
    <property name="title">
     <string>Checking Tools</string>
    </property>
    <addaction name="actionQA"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUpdatedProject"/>
    <addaction name="actionInstall_Plugins"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAssets"/>
   <addaction name="menuConfig"/>
   <addaction name="menuChecking_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport_Module">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/arrow_6.png</normaloff>:/Project/arrow_6.png</iconset>
   </property>
   <property name="text">
    <string>Import Module</string>
   </property>
  </action>
  <action name="actionRemove_Module">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/arrow_12.png</normaloff>:/Project/arrow_12.png</iconset>
   </property>
   <property name="text">
    <string>Remove Module</string>
   </property>
  </action>
  <action name="actionImport_Asset">
   <property name="icon">
    <iconset>
     <normaloff>:/Project/Add.png</normaloff>:/Project/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add Items</string>
   </property>
  </action>
  <action name="actionRemove_Items">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Delete.png</normaloff>:/Project/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Items</string>
   </property>
  </action>
  <action name="actionCreate_Workflow">
   <property name="icon">
    <iconset>
     <normaloff>:/Project/Add.png</normaloff>:/Project/Add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Edit.png</normaloff>:/Project/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Refresh.png</normaloff>:/Project/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </action>
  <action name="actionTo_Left">
   <property name="text">
    <string>to Left</string>
   </property>
  </action>
  <action name="actionTo_Right">
   <property name="text">
    <string>to Right</string>
   </property>
  </action>
  <action name="actionQA">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Edit.png</normaloff>:/Project/Edit.png</iconset>
   </property>
   <property name="text">
    <string>QA</string>
   </property>
  </action>
  <action name="actionAsset_Tracking">
   <property name="text">
    <string>Asset Tracking</string>
   </property>
  </action>
  <action name="actionUpdatedProject">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Import.png</normaloff>:/Project/Import.png</iconset>
   </property>
   <property name="text">
    <string>Updated Tools</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionInstall_Plugins">
   <property name="text">
    <string>Install Plugins</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/Project/About.png</normaloff>:/Project/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionCheck_in">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Check.png</normaloff>:/Project/Check.png</iconset>
   </property>
   <property name="text">
    <string>Check in</string>
   </property>
  </action>
  <action name="actionCheck_out">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Uninstall.png</normaloff>:/Project/Uninstall.png</iconset>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/Project/Add.png</normaloff>:/Project/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add file to Source Control</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Refresh.png</normaloff>:/Project/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Delete.png</normaloff>:/Project/Delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete file</string>
   </property>
  </action>
  <action name="actionAsset_Browser">
   <property name="text">
    <string>Asset Browser</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Connect to Source Control ...</string>
   </property>
  </action>
  <action name="actionUpdate_tools">
   <property name="icon">
    <iconset resource="../../../../src/developer/main/source/icons/IconResource.qrc">
     <normaloff>:/Project/Import.png</normaloff>:/Project/Import.png</iconset>
   </property>
   <property name="text">
    <string>Update tools</string>
   </property>
  </action>
  <action name="actionShow_History">
   <property name="text">
    <string>Show History</string>
   </property>
  </action>
  <action name="actionGet_the_latest_version">
   <property name="text">
    <string>Get the latest version</string>
   </property>
  </action>
  <action name="actionGet_version">
   <property name="text">
    <string>Get version</string>
   </property>
  </action>
  <action name="actionConnect_to_Source_Control">
   <property name="text">
    <string>Connect to Source Control</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../src/developer/main/source/icons/IconResource.qrc"/>
 </resources>
 <connections/>
</ui>
